pkgs/servers/metabase/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, lib, fetchurl, makeWrapper, jre }:
```

Comment:
The `pkgs` parameter looks unused.  Note also that `lib` can be (and is typically) accessed via `stdenv.lib`.